frontend/src/components/auth/Profile.jsx

```diff
@@ -78,6 +81,9 @@ const Profile = () => {
             if (interviewQuestions.length > 2) {
                 setLastQuestionCheck(interviewQuestions[currentQuestionIndex + 2])
```

Comment:
could this array access be out of bounds? would we need to check if interviewQuestions.length.length > currentQuestionIndex + 2?